mobify/platform-scaffold
# Fix webpack style path

## Changes
- Fix webpack style path to work with SDK components. 

## How to test-drive this PR
- `npm run dev`
- Preview [Merlin's Potions](http://localhost:3000/potions/eye-of-newt.html)
- Make sure that everything working fine and looks good.
